sample-tests/src/test/resources/sampleweb.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<web-app xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://java.sun.com/xml/ns/javaee" xsi:schemaLocation="http://java.sun.com/xml/ns/javaee http://java.sun.com/xml/ns/javaee/web-app_3_0.xsd" id="WebApp_ID" version="3.0">
+     
+  <listener>
+    <listener-class>sample.customFunction.remote.MyBigdataRDFContextListener</listener-class>
```

Comment:
Yes, should be able to set the `SystemProperties.JETTY_OVERRIDE_WEB_XML` property in your main and achieve the same result.  See https://github.com/SYSTAP/bigdata/pull/265/